mu4/notation/internal/notationparts.cpp

```diff
@@ -199,6 +199,36 @@ void NotationParts::setPartName(const ID& partId, const QString& name)
     m_partsChanged.notify();
 }
 
+void NotationParts::setPartSharpFlat(const ID& partId, const Ms::PreferSharpFlat& sharpFlat)
```

Comment:
PreferSharpFlat

I don't like this name. Better simply SharpFlat

Reply:
fixed